### PR TITLE
Updated branch policy

### PR DESCRIPTION
**Pull request by evildmp**
[ci only docs]

### CODE REVIEW

File: docs/contributing/development-policies.rst [hunk 1]
```diff
@@ -94,18 +94,16 @@ Branches
...
-    ``release/x.y.z``.
+    The default target branch for on-going development.
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI. ``develop`` will be merged into ``release/x.y.z``
+    regularly.
 
-    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
-    substantial team co-ordination.
+``feature`` branches
```
Comment (czpython) [inline docs/contributing/development-policies.rst]: we don't really use this.
most development happens on forks outside of core

Comment (vxsx) [reply docs/contributing/development-policies.rst]: don't think we need feature branches really at the moment. if only to show development intent to whoever is watching but I kinda doubt anyone is doing that really

**Comment by evildmp** [comment]
@czpython updated

File: docs/contributing/development-policies.rst [hunk 2]
```diff
@@ -94,18 +94,12 @@ Branches
...
-
 ``develop``
-    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
-    ``release/x.y.z``.
+    The default target branch for on-going development.
 
-    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
-    substantial team co-ordination.
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI. ``develop`` will be merged into ``release/x.y.z``
```
Comment (czpython) [inline docs/contributing/development-policies.rst]: `develop` is/should never be merged into `release/x.y.z`, instead commits are cherry-picked

**Comment by FinalAngel** [comment]
@evildmp can you adapt this PR so we can merge it to 3.7?

**Pull request merged**